protoc-gen-tonic/Cargo.toml

```diff
@@ -20,15 +20,15 @@ exclude = [
 [dependencies]
 heck = "0.4.0"
 prettyplease = "0.1.9"
-prost-build = { version = ">=0.11.0, <0.11.2", default-features = false }
+prost-build = { version = "0.11.4", default-features = false }
 prost-types = { version = "0.11.0", default-features = false }
 prost = { version = "0.11.0", default-features = false, features = ["std"] }
 protoc-gen-prost = { version = "0.2.0", path = "../protoc-gen-prost" }
 regex = { version = "1.5.5", default-features = false }
 syn = { version = "1.0.89", features = ["parsing", "full"] }
-tonic-build = { version = "0.8", features = [] }
+tonic-build = { version = "0.8.4", features = [] }
```

Comment:
I explicitly set this to `0.8.4`. what do you think? I dont think its strictly necessary right?

Reply:
Because we'd need a minimum of `0.8.4` in order to support the no transport functionality, this is indeed required. Otherwise someone could unintentionally pull a lower version and end up with a plugin that wouldn't compile for lack of an expected API in this dependency.